wowchemy/assets/js/wowchemy-slides.js

```diff
@@ -0,0 +1,150 @@
+import * as params from "@params";
+
+import {fixMermaid} from './wowchemy-utils';
+
+// Enable core slide features.
+enabledPlugins = [
+  RevealMarkdown,
+  RevealHighlight,
+  RevealSearch,
+  RevealNotes,
+  RevealMath,
+  RevealZoom,
+];
+
+const isObject = function (o) {
+  return o === Object(o) && !isArray(o) && typeof o !== "function";
+};
+
+const isArray = function (a) {
+  return Array.isArray(a);
+};
+
+const toCamelCase = (s) => {
+  return s.replace(/([-_][a-z])/gi, ($1) => {
+    return $1.toUpperCase().replace("-", "").replace("_", "");
+  });
+};
+
+const keysToCamelCase = function (o) {
+  if (isObject(o)) {
+    const n = {};
+
+    Object.keys(o).forEach((k) => {
```

Comment:
You are defining functions using the arrow notation `(...) =>` in some places and `function (...)` in other places. Please be consistent and use the newer arrow notation when defining functions.

Reply:
found middle ground for this .. since I see that you are using function in other files .. I followed the following rule of thumb:
- Use function when defining reusable functions
- Use arrow function for anonymous functions used in forEach and other methods that take a function as an argument